README.md

```diff
@@ -704,6 +704,7 @@ Table of Contents
 
 ## Monitoring
 
+  * [Airbrake.io](https://www.airbrake.io/) - Get started with <strong><i>Free Forever</i></strong> Error and Performance Monitoring - <strong><i>no credit card required</i></strong>! After the Free Trial period ends, you can choose to upgrade to a paid Airbrake plan, or automatically convert to a <strong><i>FREE FOR LIFE</i></strong> Airbrake Dev Account that includes (1) User, (1) Team, (2) Projects, 7,500 Errors per month, 50,000 Performance Events, and 7 days data retention.
```

Comment:
Thanks. Let’s drop the bold text etc. you will note other entries don’t have such emphasis. 

Everyone knows on a list of free things that things are free, no holds needed.